xla/service/service_executable_run_options.h

```diff
@@ -56,9 +60,17 @@ class ServiceExecutableRunOptions {
                : Status(absl::StatusCode::kUnimplemented, "No stream cache");
   }
 
+  StatusOr<StreamPool::Ptr> BorrowStream(
+      int device_ordinal, stream_executor::StreamPriority priority) const {
+    return borrow_stream_with_priority_
+               ? borrow_stream_with_priority_(device_ordinal, priority)
+               : Status(absl::StatusCode::kUnimplemented, "No stream cache");
+  }
+
  private:
   ExecutableRunOptions run_options_;
   StreamBorrower borrow_stream_;
+  StreamBorrowerWithPriority borrow_stream_with_priority_;
```

Comment:
If we unify, we will need just one borrower, and we can specify both device ordinal and priority.

Reply:
removed this